recipes/default.rb

```diff
@@ -18,3 +18,5 @@
 #
 
 include_recipe 'rdiff-backup::client'
+
+package 'rdiff-backup'
```

Comment:
IMO we should remove the recipe inclusion above and have the default recipe *only* install the rdiff-backup package. You should move the yum recipe inclusions in client over to here instead.